### PR TITLE
scatter_add uses src, not other

### PR DESCRIPTION
**Pull request by ssnl**
using `other` kwarg gives `TypeError: scatter_add_() missing 1 required positional arguments: "src"`

### CODE REVIEW

**Comment by SsnL** [comment]
ping

**Review by zou3519 — APPROVED** [review]

File: torch/_tensor_docs.py [hunk 1]
```diff
@@ -2418,23 +2418,23 @@ def callable(a, b) -> number
 
 add_docstr_all('scatter_add_',
                r"""
-scatter_add_(dim, index, other) -> Tensor
+scatter_add_(dim, index, src) -> Tensor
```
Comment (zou3519) [inline torch/_tensor_docs.py]: "scatter_add" is also wrong... https://pytorch.org/docs/stable/tensors.html#torch.Tensor.scatter_add (it takes src but the docs lists it as "source")

Comment (SsnL) [reply torch/_tensor_docs.py]: fixed below :)

Comment (zou3519) [reply torch/_tensor_docs.py]: Oho, I didn't read the entire PR. Thanks!! (and feel free to ping again for review earlier in the future, the notification for this escaped my radar)

**Pull request closed without merging**